snap/snapcraft.yaml

```
@@ -5,8 +5,78 @@ summary: Snap for NVIDIA DCGM and DCGM exporter
 license: Apache-2.0
 contact: solutions-engineering@lists.canonical.com
 description: |
-  This snap includes NVIDIA DCGM and DCGM exporter to provide GPU monitoring via
-  Prometheus metrics. This snap does not include the dcgmproftester.
+  This snap includes NVIDIA DCGM and DCGM-exporter to monitor GPU via Prometheus metrics
```

I think this sentence focuses too much on the metrics, which can be odd since the snap name is after all DCGM and not DCGM Exporter
```suggestion
  This snap includes NVIDIA DCGM and DCGM-exporter to manage and monitor NVIDIA GPUs, either directly via the CLI or via Prometheus metrics.
```